drive/types.bal

```diff
@@ -475,18 +475,18 @@ type CreateFileOptional record {
     boolean useContentAsIndexableText?; 
 };
 
-# Update file optional parameters
+# Update file optional parameters.
 #
-# + ocrLanguage - A language hint for OCR processing during image import (ISO 639-1 code).
-# + removeParents - A comma-separated list of parent IDs to remove. 
+# + ocrLanguage - A language hint for OCR processing during image import (ISO 639-1 code)
+# + removeParents - A comma-separated list of parent IDs to remove
 # + keepRevisionForever - Whether to set the 'keepForever' field in the new head revision. 
```

Comment:
Replace "Whether" with "Indicates whether".

Reply:
Hi,
We have used the same content from [1] for parameter descriptions
[1]https://developers.google.com/drive/api/v3/reference/files
Thanks